test/lib/client.js

```diff
@@ -1,34 +1,33 @@
-"use strict";
+'use strict';
+
 // webdriver deps
-var WebdriverIO = require('webdriverio');
-var Promise = require('rsvp').Promise; // jshint ignore:line
-var paginationUtils;
-var config = require('config');
-var options = {
-    desiredCapabilities: {
-        browserName: config.get('browserName')
-    },
-    host: config.get('host'),
-    port: config.get('port')
+const WebdriverIO = require('webdriverio');
+const config = require('config');
+
+const options = {
+  desiredCapabilities: {
+    browserName: config.get('browserName'),
+  },
+  host: config.get('host'),
+  port: config.get('port'),
 };
-var client = WebdriverIO.remote(options);
-paginationUtils = require('./../../util/pagination.js')(client);
+const client = WebdriverIO.remote(options);
+
+require('./../../util/pagination.js')(client);
 
-var clientReady = new Promise(function (resolve, reject) {
-    client.init(function(err) {
-        if(!err) {
-            resolve();
-        } else {
-            reject(err);
-        }
-    });
-}).catch(function(err) {
-    console.error(err);
-    throw err;
+const clientReady = new Promise((resolve, reject) => {
```

Comment:
4reals. Haven't updated Webdriver.io _yet_, which might make this totally unnecessary.